### PR TITLE
Port ToolTips to md file

### PR DESCRIPTION
**Pull request by benfleuty**
This PR introduces docs/SETTINGS.md which lists the tool tips provided in code.
Tagging issue https://github.com/soda3x/ArmaReforgerServerTool/issues/46 for visibility to affected by the issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ Create Arma Reforger Dedicated Servers on Windows with this easy to use GUI tool
 ## Requirements
 - [dotNET 6 Runtime](https://dotnet.microsoft.com/en-us/download)
 
+## Discord
+
+Come discuss with fellow users, seek help etc. on the Discord server [here](https://discord.gg/RmTH7gKdfK)
+
+## Settings
```
Comment (soda3x) [inline README.md:21]: Maybe change to "Server Parameters"

File: README.md [hunk 2]
```diff
@@ -14,6 +14,13 @@ Create Arma Reforger Dedicated Servers on Windows with this easy to use GUI tool
...
+See [here](docs/SETTINGS.md) for more information on the settings in the app.
```
Comment (soda3x) [inline README.md:22]: settings -> Server Parameters

**Review by soda3x — CHANGES_REQUESTED** [review]
Thanks a lot for this! I have two minor actions, let me know if you agree :)

File: README.md [hunk 3]
```diff
@@ -14,6 +14,13 @@ Create Arma Reforger Dedicated Servers on Windows with this easy to use GUI tool
 ## Requirements
 - [dotNET 6 Runtime](https://dotnet.microsoft.com/en-us/download)
 
+## Discord
```
[the remark this answers is not in the document]
Comment (benfleuty) [reply README.md:17]: Will do. Where'd you like server parameters?

[the remark this answers is not in the document]
Comment (soda3x) [reply README.md:17]: Just above Discord is fine

File: README.md (hunk 1 above)
Comment (benfleuty) [reply README.md]: Done in https://github.com/soda3x/ArmaReforgerServerTool/pull/50/commits/04f04a46a2926a00fe23faa315bb240ade23a2cf?notification_referrer_id=NT_kwDOAVH0KrQxMDI5OTY4MTU1NjoyMjE0ODEzOA

File: README.md (hunk 2 above)
Comment (benfleuty) [reply README.md]: Done in https://github.com/soda3x/ArmaReforgerServerTool/pull/50/commits/04f04a46a2926a00fe23faa315bb240ade23a2cf

File: README.md [hunk 4]
```diff
@@ -30,3 +23,10 @@ See [here](docs/SETTINGS.md) for more information on the settings in the app.
 ![Screenshot](https://github.com/soda3x/ArmaReforgerServerTool/raw/main/servertoolscreen4.png)
 
 ![Screenshot](https://github.com/soda3x/ArmaReforgerServerTool/raw/main/servertoolscreen2.png)
+
+## Discord
+
+Come discuss with fellow users, seek help etc. on the Discord server [here](https://discord.gg/RmTH7gKdfK)
+
+## Server Paramters
```
Comment (soda3x) [inline README.md:31]: Small typo

**Review by soda3x — CHANGES_REQUESTED** [review]
One more comment, just a small typo!

File: README.md (hunk 4 above)
Comment (benfleuty) [reply README.md]: Done in https://github.com/soda3x/ArmaReforgerServerTool/pull/50/commits/101c683f9ee850fea381138687373d08f27d07c7.

**Review by soda3x — APPROVED** [review]

**Pull request merged**